controller/controller.go

```diff
@@ -106,30 +96,33 @@ func New(config Config) (*Controller, error) {
 	if len(config.ResourceSets) == 0 {
 		return nil, microerror.Maskf(invalidConfigError, "%T.ResourceSets must not be empty", config)
 	}
-
-	if config.BackOffFactory == nil {
-		config.BackOffFactory = func() backoff.Interface { return backoff.NewMaxRetries(7, 1*time.Second) }
+	if config.RuntimeObjectFactory == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.RuntimeObjectFactory must not be empty", config)
 	}
+
 	if config.Name == "" {
 		return nil, microerror.Maskf(invalidConfigError, "%T.Name must not be empty", config)
 	}
+	if config.ResyncPeriod == 0 {
+		return nil, microerror.Maskf(invalidConfigError, "%T.ResyncPeriod must not be empty", config)
```

Comment:
Doesn't defaulting to some value make sense?

Reply:
Which value do you think should it be?

Reply:
I think it makes sense to start with `5m`. Wasn't that previous default?

Reply:
It is. Sorted.